Dockerfile

```diff
@@ -0,0 +1,28 @@
+ARG PYTHON_VERSION=3.6
```

Comment:
move to `.circleci/Dockerfile`

Reply:
Useful change! Will apply this across all addons, after this is approved/merged